recipes/azure-purview-account/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "azure-purview-account" %}
+{% set version = "1.0.0b1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/azure-purview-account-{{ version }}.zip
+  sha256: d9a02494a62cfb5be2cf5de2b64becd23c5949cf05856d7dfe665256d5ba20ae
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - azure-core <2.0.0,>=1.16.0
+    - msrest >=0.6.21
+    - six >=1.11.0
+    - azure-purview-nspkg  # [py<30]
+    - typing  # [py<35]
```

Comment:
I don't understand this. It depends on `azure-purview-nspkg` for Python < 3.0, but azure-purview-nspkg is only installable for Python >=3.6?

I'll note for both of these, it's much preferrable to unconditionally depend on things to make the recipe noarch. Also, conda-forge in general is only trying to support Python >=3.7 these days.

Reply:
I don't understand either. The recipe was created using grayskull. The setup.py of azure-purview-account does state the dependencies as in the meta.yaml. I propose to leave it as is, or drop both lines since there is no support for Python < 3.7.

Reply:
Okay, after thinking a bit more. Since for python 2 the dependencies can't resolve, we should not support py<30 at all.

We can also unconditionally depend on typing and make it noarch. I'll implement the changes as described here. Let me know if you have a different preference.